CMakeLists.txt

```diff
@@ -2337,7 +2338,16 @@ endif ()
 
 if (is_multiconfig)
   # Rippled headers. Only useful for IDEs.
```

Comment:
I think this comment doesn't apply any more and needs updating - maybe something like "This code finds all source files in the src subdirectory for inclusion in the IDE file tree as non-compiled sources. Since this file list will have some overlap with files we have already added to our targets to be compiled, we explicitly remove any of these target source files from this list."